### PR TITLE
slide calculator over with expanded pane [issue #48]

### PR DESCRIPTION
**Pull request by japamat0**
Fix for issue: [#48](https://github.com/desmosinc/gifsmos/issues/48)

I added a container for the `<App>`, connecting it to the `ui.expandedPane` in the store. This allowed me to use ClassNames from the classnames library to add classes based on which pane is shown, allowing me to move the calculator over to account for the expanded pane:

Screen Shot 2019-06-24 at 4 05 14 PM

### CODE REVIEW

**Comment by japamat0** [comment]
Closing, prematurely submitted PR

File: src/components/App.css [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
   position: absolute;
   top: 40px;
   left: 70px;
   right: 0;
   bottom: 0;
 }
+
+.calculator-burst {
+  left: 271px;
+  animation: fade-in 0.5s;
```
Comment (misscoded) [inline src/components/App.css]: Per offline discussion, remote `fade-in` animations for all classes and combine `.calculator-burst` `.calculator-settings` together,

**Review by misscoded — CHANGES_REQUESTED** [review]
See comment, but also note:

It appears as though the same positioning intent can be achieved with less `z-index` changes: namely, keep the `z-index: 2` for the sidebar, but set the `.Preview-expanded` to `z-index: 1`, as well.

File: src/components/App.css (hunk 1 above)
Comment (japamat0) [reply src/components/App.css]: Done. Grouped like classes together, removed animations, changed z-indexes per above comment

File: src/components/App.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react';
+import ClassNames from 'classnames';
```
Comment (misscoded) [inline src/components/App.js]: This is named `classNames` in all other files -- follow that pattern.

File: src/components/Burst.css [hunk 3]
```diff
@@ -1,5 +1,6 @@
 .Burst {
   position: absolute;
+  z-index: 2;
```
Comment (misscoded) [inline src/components/Burst.css]: Again, remove all of these component-specific `z-index` with the exception of Preview, whose changes suggested previously should be kept.

**Review by misscoded — CHANGES_REQUESTED** [review]

File: src/components/App.js (hunk 2 above)
Comment (japamat0) [reply src/components/App.js]: Done

**Review by misscoded — APPROVED** [review]

**Comment by misscoded** [comment]
:shipit:

**Pull request merged**